ui/pages/onboarding-flow/privacy-settings/privacy-settings.test.js

```diff
@@ -89,35 +104,6 @@ describe('Privacy Settings Onboarding View', () => {
     expect(setUseAddressBarEnsResolutionStub.mock.calls[0][0]).toStrictEqual(
       false,
     );
-
-    // toggle back to true
```

Comment:
Removed because there's no logic difference to turn and off while using the same setter